app/assets/javascripts/application.js

```diff
@@ -12,41 +12,42 @@
 //
 
 
-//= require leaflet/dist/leaflet-src.js
+// = require leaflet/dist/leaflet-src.js
```

Comment:
Hi @alaxalves this is re-introducing these spaces which cause these JS assets not to be compiled in. If this is intended, could you explain why? Just seeing this was recently undone by @kaustubh-nair and I feel like we may be crossing wires communication-wise here. Thanks!!! 👍

Reply:
Actually it was Rubocop's `rubocop --auto-fix` task. I will undo this. Thx for pointing it out!

Reply:
@jywarren Just fixed this! Could you check it out again please?

Reply:
aha! Rubocop! 😄 Thank you!